Modules/mathmodule.c

```diff
@@ -1072,19 +1072,22 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 FUNC1(acos, acos, 0,
       "acos($module, x, /)\n--\n\n"
-      "Return the arc cosine (measured in radians) of x.")
+      "Return the arc cosine (measured in radians) of x. "
+      "The result is between 0 and pi")
```

Comment:
One nitpick: please could you add full stops to the end of the sentences (here and for `asin` and `atan`, too)?